### ISSUE TITLE
ImportOrder: checkSeparatorInGroup works wrong when separated=false

### ISSUE THREAD

**Issue by djydewang** [checkstyle/checkstyle#5090]
```
$ cat TestClass.java
import java.awt.Button;

import java.awt.Dialog;

import java.awt.Frame;
import java.awt.event.ActionEvent;
import javax.swing.JComponent;

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
    <module name="ImportOrder">
      <property name="groups" value="/^java\./,javax,org"/>
      <property name="ordered" value="true"/>
      <property name="separated" value="false"/>
    </module>
    </module>
</module>

$ java -jar checkstyle-8.2-all.jar -c TestConfig.xml TestClass.java
Starting audit...
Audit done.
```

There is no warning message and I excepted:
```
Extra separation in import group before 'java.awt.Dialog'.
Extra separation in import group before 'java.awt.Frame'.
```

**Comment by rnveach** [comment]
> 進口組以前的額外分離 'java.awt.Dialog'
> 進口組以前的額外分離 'java.awt.Frame'

This text reads in English as:
`Extra separation in import group before 'java.awt.Dialog'.`
`Extra separation in import group before 'java.awt.Frame'.`

**Comment by romani** [comment]
@djydewang , can you find what how to change command line to print messages in English ? 
we need to update our issue template to avoid non English output, as non of us speak all languages we support in localization.

**Comment by romani** [comment]
this looks similar to #5065

**Comment by djydewang** [comment]
@romani I have finded [cause](https://github.com/checkstyle/checkstyle/blob/08793f2816a2ce275afdb89f2a39fa535e0cf3e6/src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java#L467) of this issue.
```
private boolean checkSeparatorInGroup(int groupIdx, boolean isStatic, int line) {
        return !beforeFirstImport && separated && groupIdx == lastGroup
                && isStatic == lastImportStatic && line - lastImportLine > 1;
    }
```
variable `separated` means  whether or not groups of imports must be separated from one another
by at least one blank line.So it should not be in Judgment expression of checkSeparatorInGroup.`separated` is initialized as false,and if I removed it from expression,checkSeparatorInGroup would worked strictly than befor,So many test case should be refactored.
Should I do that?

**Comment by romani** [comment]
If tests are incorrect - they have to refactored.

**Comment by romani** [comment]
@djydewang , 
Please do me favor and recheck if following format on commanline will print error messages in English:
`java -Duser.language=en -Duser.country=US ...`

**Comment by djydewang** [comment]
It works right.

**Comment by romani** [comment]
I updated issue template, to avoid such situations in future.

**Comment by romani** [comment]
@djydewang , please update issue description to have English output.

**Comment by djydewang** [comment]
Done.

**Comment by romani** [comment]
looks like already resolved, most likely by #5065.
```
$ java -jar checkstyle-8.8-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /var/tmp/Test.java:3: Extra separation in import group before 'java.awt.Dialog' [ImportOrder]
[ERROR] /var/tmp/Test.java:5: Extra separation in import group before 'java.awt.Frame' [ImportOrder]
Audit done.
Checkstyle ends with 2 errors.
```
